specification/msi/resource-manager/Microsoft.ManagedIdentity/2015-08-31-preview/ManagedIdentity.json

```diff
@@ -0,0 +1,544 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-08-31-preview",
```

Comment:
Why is the version 2015?

Reply:
Right now we are generating the Swagger and tools for our private preview version. We are planning to keep the version name same for private preview. Next year for GA, we will change the version